.github/workflows/download-translations.yml

```diff
@@ -0,0 +1,137 @@
+name: 'Download translations from Crowdin'
+
+permissions:
+  actions: write
+  checks: write
+  contents: write
+  deployments: write
+  pull-requests: write
+  statuses: write
+  
+  
+on:
+  repository_dispatch:
+    types:
+      - file-fully-reviewed
+
+jobs:
+  synchronize-with-crowdin:
+    runs-on: ubuntu-latest
+    if: github.repository == 'binary-com/deriv-app'
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+          ref: master
+
+      - name: Get cached dependencies
+        id: cache_npm
+        uses: actions/cache@v3
+        with:
+          key: translations-build-${{ hashFiles('**/package-lock.json') }}
+          path: |
+              node_modules
+              packages/account/node_modules
+              packages/api/node_modules
+              packages/appstore/node_modules
+              packages/bot-skeleton/node_modules
+              packages/bot-web-ui/node_modules
+              packages/cashier/node_modules
+              packages/components/node_modules
+              packages/core/node_modules
+              packages/hooks/node_modules
+              packages/cfd/node_modules
+              packages/indicators/node_modules
+              packages/p2p/node_modules
+              packages/reports/node_modules
+              packages/shared/node_modules
+              packages/stores/node_modules
+              packages/trader/node_modules
+              packages/translations/node_modules
+              packages/utils/node_modules
+              packages/analytics/node_modules
+
+
+      - name: Install dependencies
+        if: ${{ steps.cache_npm.outputs.cache-hit != 'true' }}
+        run: |
+          npm run bootstrap
+
+      # In this step we're doing a couple things:
+      # - We download the latest translation files from Crowdin, if there are new files, we create a PR.
+      - name: Generate and push to Crowdin
+        run: |
+          branch_name="deriv_app_translations_2"
```

Comment:
We create a separate branch `deriv_app_translations_2` to test out the workflow first without conflicting the current translations workflow `push_and_pull_crowdin_translations.yml`